### PR TITLE
Deprecate OpsGenie integration in Sensu Enterprise <3.0

### LINKED ISSUE #525: Deprecate OpsGenie API v1
On June 30, OpsGenie will deprecate their version 1 API that is used by Sensu Enterprise prior to version 3.0. The docs for versions <3.0 should be updated to specify that the OpsGenie integration is no longer supported after that date and direct users to the upgrading guide.

## This is an issue with:
- Update (Add missing or refresh existing content)

### PR DESCRIPTION
**Pull request by apaskulin**
## Description

[According to their site](https://www.opsgenie.com/news/335), OpsGenie has shut down their v1 API, making the Sensu Enterprise OpsGenie integration unavailable in versions 2.6, 2.7, and 2.8. This PR removes the OpsGenie integration content from those versions and updates the 3.0 upgrade guide.

I wasn't able to find any follow-up communications from OpsGenie to indicate that they postponed the API shutdown, so I went with the originally stated shutdown date.

## Motivation and Context

Closes #525 

## How Has This Been Tested?

Locally with yarn

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New doc/guide
- [ ] Fixing errata
- [x] Update (Add missing or refresh existing content)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] All tests have passed.

### CODE REVIEW

File: content/sensu-enterprise/2.6/integrations/opsgenie.md [hunk 1]
```diff
@@ -8,120 +8,12 @@ menu:
...
-- [Configuration](#configuration)
-  - [Example(s)](#examples)
-  - [Integration Specification](#integration-specification)
-    - [`opsgenie` attributes](#opsgenie-attributes)
+_WARNING: To use the OpsGenie integration, please [upgrade to the latest version of Sensu Enterprise][1]._
 
-## Overview
+Following the shutdown of the OpsGenie v1 API on June 30, 2018, Sensu Enterprise 2.6 no longer supports the OpsGenie integration.
+To use the Sensu Enterprise OpsGenie integration, please [upgrade to the latest version of Sensu Enterprise][1].
+See the [Sensu Enterprise 3.0 upgrade guide][1] and the [OpsGenie integration docs][2] for more information.
```
Comment (rgeniesse) [inline content/sensu-enterprise/2.6/integrations/opsgenie.md]: As this points to `latest`, `Sensu Enterprise 3.0 upgrade guide` won't be true when a new major or minor version of Sensu Enterprise comes out. The 3.0 docs have the only upgrade guide that is relevant to the OpsGenie change currently. Or is the upgrade guide going to brought up to all versions 3.x and above? That would make `latest` continue to work (still would want to remove the 3.0 label I think) and cover future scenarios where users may be upgraded from 2.x to say 3.1.

That all said, the Enterprise user base is much smaller than Core, so maybe this isn't a large issue to begin with. :-).

Comment (apaskulin) [reply content/sensu-enterprise/2.6/integrations/opsgenie.md]: Thank you for catching this! I'll change the link to point to the 3.0 guide

Comment (apaskulin) [reply content/sensu-enterprise/2.6/integrations/opsgenie.md]: Updated all these links to point to 3.0 specifically. This emphasizes that "3.0 or later" is the key bit here and maintains consistency. In the future, the "You're viewing documentation for an older or prerelease version..." bar will tip off users that a newer version exists after navigating to the 3.0 pages.

**Review by rgeniesse — APPROVED** [review]
💯 awesome!

**Pull request merged**